### PR TITLE
Cisco Cat8k series router updates

### PR DESCRIPTION
**Pull request by hyberdk**
Add new:

- C8200-1N-4T
- C8200L-1N-4T
- C8300-1N1S-4T2X
- C8300-2N2S-6T

updates:

- C8500-12X
- C8500L-8S4X

### CODE REVIEW

File: device-types/Cisco/C8200-1N-4T.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    type: rj-45
+  - name: usb
+    type: usb-a
+power-ports:
+  - name: P0
+    type: iec-60320-c14
+    maximum_draw: 150
+  - name: P1
+    type: iec-60320-c14
+    maximum_draw: 150
```
Comment (DanSheps) [inline device-types/Cisco/C8200-1N-4T.yaml:24]: I would model this either as a module or as a DC Terminal (or Molex if that is available)

File: device-types/Cisco/C8200L-1N-4T.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    type: rj-45
+  - name: usb
+    type: usb-a
+power-ports:
+  - name: P0
+    type: iec-60320-c14
+    maximum_draw: 150
+  - name: P1
+    type: iec-60320-c14
+    maximum_draw: 150
```
Comment (DanSheps) [inline device-types/Cisco/C8200L-1N-4T.yaml:24]: See comment above

File: device-types/Cisco/C8200-1N-4T.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+rear_image: true
+weight: 10
+weight_unit: lb
+console-ports:
+  - name: con 0
+    type: rj-45
+  - name: usb
+    type: usb-a
+power-ports:
+  - name: P0
```
Comment (DanSheps) [inline device-types/Cisco/C8200-1N-4T.yaml:19]: We should have a consistent naming convention.  I would go with "PSU"

File: device-types/Cisco/C8200L-1N-4T.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+rear_image: true
+weight: 10
+weight_unit: lb
+console-ports:
+  - name: con 0
+    type: rj-45
+  - name: usb
+    type: usb-a
+power-ports:
+  - name: P0
```
Comment (DanSheps) [inline device-types/Cisco/C8200L-1N-4T.yaml:19]: We should have a consistent naming convention.  I would go with "PSU"

File: device-types/Cisco/C8300-1N1S-4T2X.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    type: rj-45
+  - name: usb
+    type: usb-micro-b
+power-ports:
+  - name: PSU0
+    type: iec-60320-c14
+    maximum_draw: 650
+  - name: PSU1
+    type: iec-60320-c14
+    maximum_draw: 650
```
Comment (DanSheps) [inline device-types/Cisco/C8300-1N1S-4T2X.yaml:25]: These should be modelled as module bays

File: device-types/Cisco/C8300-2N2S-6T.yaml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    type: rj-45
+  - name: usb
+    type: usb-micro-b
+power-ports:
+  - name: PSU0
+    type: iec-60320-c14
+    maximum_draw: 650
+  - name: PSU1
+    type: iec-60320-c14
+    maximum_draw: 650
```
Comment (DanSheps) [inline device-types/Cisco/C8300-2N2S-6T.yaml:25]: These should be modelled as module bays

File: device-types/Cisco/C8500-12X.yaml
Comment (DanSheps) [inline device-types/Cisco/C8500-12X.yaml:1]: The power should be modelled as a module

**Review by DanSheps — CHANGES_REQUESTED** [review]

File: device-types/Cisco/C8200-1N-4T.yaml (hunk 1 above)
Comment (hyberdk) [reply device-types/Cisco/C8200-1N-4T.yaml]: added as module

File: device-types/Cisco/C8200L-1N-4T.yaml (hunk 2 above)
Comment (hyberdk) [reply device-types/Cisco/C8200L-1N-4T.yaml]: added as module

File: device-types/Cisco/C8300-2N2S-6T.yaml (hunk 6 above)
Comment (hyberdk) [reply device-types/Cisco/C8300-2N2S-6T.yaml]: done as module

File: device-types/Cisco/C8500-12X.yaml
Comment (hyberdk) [reply device-types/Cisco/C8500-12X.yaml:1]: done as module

File: device-types/Cisco/C8300-1N1S-4T2X.yaml (hunk 5 above)
Comment (hyberdk) [reply device-types/Cisco/C8300-1N1S-4T2X.yaml]: done as modules

**Comment by hyberdk** [comment]
> Please remove the pickle files from your PR. These should not be committed/modified manually

they are already deleted in afa56ca21d9310e90d5ffe7d3059ee5349520b50 or what?

**Comment by danner26** [comment]
image
These files should not be modified

**Pull request closed without merging**